src/components/ListView/ListViewItem.js

```diff
@@ -58,15 +61,42 @@ class ListViewItem extends React.Component {
               additionalInfo={additionalInfo}
               actions={actions}
             />
-          </ListViewGroupItemHeader>
-          <ListViewGroupItemContainer
-            expanded={expanded}
-            onClose={() => this.toggleExpanded()}
-          >
-            {children}
-          </ListViewGroupItemContainer>
-        </ListViewGroupItem>
-      );
+            <ListViewGroupItemContainer
+              expanded={compoundExpanded}
+              onClose={onCloseCompoundExpand}
+            >
+              {children}
+            </ListViewGroupItemContainer>
+          </ListViewGroupItem>
+        );
+      } else {
+        return (
+          <ListViewGroupItem expanded={expanded} stacked={stacked} {...other}>
+            <ListViewGroupItemHeader
+              toggleExpanded={() => this.toggleExpanded()}
```

Comment:
i didn't catch this on previous reviews... can be simplified to `toggleExpanded={ this.toggleExpanded}`